lib/pltr/v2/reducers/root.js

```diff
@@ -21,6 +27,18 @@ const root = (dataRepairers) => (state, action) => {
   const isSeries = action.type.includes('@@') ? false : isSeriesSelector(state)
   const mainReducer = unrepairedMainReducer(dataRepairers)
   switch (action.type) {
+    case ADD_BOOK_FROM_TEMPLATE:
+      // cards from the template need to know the new ids of lines and beats from the template
+      // the strategy here is to use the state's next id value + the template id's current value
+      // the card reducer will have access to the state's next id value so it will be able to determine the correct id
```

Comment:
Great comment!  Was about to mention that we needed something like this when I found it! :) :+1: